ext/em.cpp

```diff
@@ -867,17 +870,23 @@ SelectData_t::SelectData_t()
 	rb_fd_init (&fderrors);
 }
 
+SelectData_t::~SelectData_t()
+{
+	rb_fd_term (&fdreads);
+	rb_fd_term (&fdwrites);
+	rb_fd_term (&fderrors);
```

Comment:
These `rb_fd_term` will cleanup heap allocated memory associated with the rb_fdset_t, but I wanted to avoid allocation/deallocation on every reactor tick so I made the entire SelectData_t get allocated once when the reactor is created.